src/redux/reducers/siloDomainData/editEntries.js

```diff
@@ -420,6 +434,89 @@ const setEntryCommentsCount = (state, action) => {
     return newState;
 };
 
+const setEntriesVerificationStatus = (state, action) => {
+    const {
+        leadId,
+        entries: entriesFromServer,
+    } = action;
+
+    const {
+        editEntries: {
+            [leadId]: {
+                entries = [],
+            } = {},
+        } = {},
+    } = state;
```

Comment:
For these kind of access, we can just use optional chaining now.


```suggestion
const entries = state.?editEntries.?[leadId].?entries ?? [];
```